pkgs/top-level/all-packages.nix

```diff
@@ -18667,7 +18667,8 @@ with pkgs;
   texinfo6_5 = callPackage ../development/tools/misc/texinfo/6.5.nix { }; # needed for allegro
   texinfo6_7 = callPackage ../development/tools/misc/texinfo/6.7.nix { }; # needed for gpm, iksemel and fwknop
   texinfo6 = callPackage ../development/tools/misc/texinfo/6.8.nix { };
-  texinfo = texinfo6;
+  texinfo7 = callPackage ../development/tools/misc/texinfo/7.0.nix { };
```

Comment:
Minor note: I think the use of versioned files (and attributes) is not really needed for `texinfo`. I would suggest renaming `7.0.nix` to `default.nix` and using it as a `texinfo` attribute without introduction of `texinfo7`.

Reply:
Yes, texinfo is used by several packages.
Regarding "versioned files (and attributes)" I followed the same style that exists currently; happy to change it if folks agree that is an improvement or how it should be done.

Reply:
@trofi if you have strong preference for your suggestion to be considered please make it be known. I haven't heard from other folks having an opinion about this. Happy to make the change if there is enough merit or interest, otherwise I'd personally keep it as is.

Reply:
It's not a strong suggestion. I don't mind keeping it as is.

Reply:
Alright, thanks for the response, @trofi 👍